build.gradle

```diff
@@ -66,6 +66,6 @@ task releng {
 // ./gradlew bintrayUpload
 
 task wrapper(type: Wrapper) {
-  gradleVersion = "4.4"
+  gradleVersion = "4.4.1"
```

Comment:
in order to use Gradle 4.5+, we first need to move away from https://github.com/novoda/bintray-release
https://github.com/novoda/bintray-release/issues/177